main.tex

```diff
@@ -1162,10 +1162,133 @@ \subsection{Monotone operators}
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\section{Time dependent problems}
+\section{Time-dependent problems}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+We now study time-dependent problems, where the solutions are functions $V\ni u:[0,T]\times \Omega\to \R$. In this section, we consider $\mathcal{L}:V\to V'$ an elliptic operator. 
+
+\begin{definition}[Parabolic and hyperbolic generic operators]
+    Let $\partial_t$ and $\partial_{tt}$ represent the first and second order time derivative operators, respectively. Then, we say that
+    \begin{align*}
+        \partial_t + \mathcal{L} &\qquad \text{is parabolic,}\\
+        \partial_{tt} + \mathcal{L} &\qquad \text{is hyperbolic.}
+    \end{align*}
+\end{definition}
+The fundamental difference between parabolic and hyperbolic systems is the behaviour of their \textit{energy} $E(t)$. Let us formally derive this fact.
+
+\begin{enumerate}
+    \item Parabolic systems: we can readily write the weak form of the parabolic equation $(\partial_t + \mathcal{L})u=0$ as
+    \begin{equation*}
+        (\partial_t u, v) + (\mathcal{L}u, v) = 0 \qquad \forall v\in V.
+    \end{equation*}
+    In particular, we set $v=u$, and we obtain
```

Comment:
Agrégale porfa una pequeña aclaración: 
"We note that this weak formulation seems unbalanced as it is being tested only on the space variable. We will accept this for now, but see later on that it is indeed a good weak formulation for analyzing the problem using our knowledge of elliptic (and G\o{a}rding) operators."